src/plugins/console/public/application/containers/editor/monaco/monaco_editor.tsx

```diff
@@ -74,6 +74,10 @@ export const MonacoEditor = ({ initialTextValue }: EditorProps) => {
     return actionsProvider.current!.getDocumentationLink(docLinkVersion);
   }, [docLinkVersion]);
 
+  const autoIndentCallback = useCallback(async (event: React.MouseEvent) => {
```

Comment:
nit: the argument "event" is now unused and can be deleted

Reply:
Nice catch! I removed it in [3e1264f](https://github.com/elastic/kibana/pull/181613/commits/3e1264f4f8718c48d1c590192c5115afca3f0bb7).